conda/base/context.py

```diff
@@ -511,8 +511,18 @@ def default_channels(self):
     def custom_multichannels(self):
         from ..models.channel import Channel
 
+        if self._default_channels:
+            _default_channels = self._default_channels
+        else:
+            _default_channels = list(DEFAULT_CHANNELS)
+            if (self.subdir == "osx-64" and int(platform.mac_ver()[0].split('.')[1]) < 11
+                    or self.bits == 32):
+                del _default_channels[2]
+            if not on_win:
+                del _default_channels[-1]
+
```

Comment:
This would prevent `default_channels` to be empty. For debugging purposes I find it sometimes useful to be able to clear that value..
Also, the `[2]`/`[-1]` indexing look rather cryptic; it would be better if one could see right away that the `mro`/`msys` are concerned.

Reply:
`platform.mac_ver` returns `('', ('', '', ''), '')` on Linux, can we abstract this away and default it to `10.9` in this case?